Source/CombatExtended/CombatExtended/Jobs/JobGiver_DefenderReloadTurret.cs

```diff
@@ -18,7 +18,7 @@ public class JobGiver_DefenderReloadTurret : ThinkNode_JobGiver
         /// How low can ammo get before we want to reload the turret?
         /// Set arbitrarily, balance if needed.
         /// </summary>
-        private const float ammoReloadThreshold = .5f;
+        public const float AmmoReloadThreshold = .5f;
```

Comment:
If the goal is to have this match the drop beacon behavior, then it should *probably* only be defined in one place.  Make it `public static const float` and then set the other value to this one.